bench/tx-generator/src/Cardano/TxGenerator/Setup/Plutus.hs

```diff
@@ -27,14 +27,19 @@ import           Cardano.Ledger.Alonzo.TxInfo (exBudgetToExUnits)
 import qualified PlutusLedgerApi.V1 as PlutusV1
 import qualified PlutusLedgerApi.V2 as PlutusV2
 
+import           Cardano.Benchmarking.PlutusScripts(findPlutusScript)
 import           Cardano.TxGenerator.Types
 
 
 type ProtocolVersion = (Int, Int)
 
 
-readPlutusScript :: FilePath -> IO (Either TxGenError ScriptInAnyLang)
-readPlutusScript fp
+readPlutusScript :: Either String FilePath -> IO (Either TxGenError ScriptInAnyLang)
+readPlutusScript (Left s)
+  = return
+  $ maybe (Left . TxGenError $ "readPlutusScript: " ++ s ++ " not found.")
+            Right (findPlutusScript s)
```

Comment:
This might be more readable, since it:
1. lowers the arity of the functions used from 3 to 2.
2. puts the expected case ahead of the error case.

```suggestion
  $ (Right <$> findPlutusScript s)
     & fromMaybe (Left . TxGenError $ "readPlutusScript: " ++ s ++ " not found.")
```